course_catalog/views.py

```diff
@@ -466,3 +466,22 @@ class PodcastEpisodesViewSet(viewsets.ReadOnlyModelViewSet):
             Prefetch("topics", queryset=CourseTopic.objects.all()),
         )
     )
+
+
+class EpisodesInPodcast(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for listing PodcastEpisodes for a given Podcast primary key
+    """
+
+    serializer_class = PodcastEpisodeSerializer
+    permission_classes = (ReadOnly & PodcastFeatureFlag,)
```

Comment:
do we need paging at all for the episodes? For instance if a podcast has 100's/1000's of episodes?

Reply:
@aliceriot looked at this and I think we're good for now, performance-wise